packages/button/docs/IconButton.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import configureIcon from "@kaizen/component-library/icons/configure.icon.svg"
+import * as React from "react"
+import { withDesign } from "storybook-addon-designs"
+import { IconButton } from ".."
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { figmaEmbed } from "../../../storybook/helpers"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.button}/Icon Button`,
+  component: IconButton,
+  args: {
+    label: "Label",
+  },
+  parameters: {
+    actions: {
+      argTypesRegex: "^on.*",
+    },
+    docs: {
+      description: {
+        component:
+          'import { IconButton } from "@kaizen/button". This IconButton supersedes "@kaizen-draft/button".',
```

Comment:
```suggestion
          'import { IconButton } from "@kaizen/button". This IconButton supersedes "@kaizen/draft-button".',
```

Haha oops neither of us noticed at the time 🙃